src/search/components/SearchResults.js

```diff
@@ -18,16 +21,16 @@ export default class SearchResults extends React.Component {
     let messageText;
 
     if (query && results.length > 0) {
-      messageText = `Your search for "${query}" returned ${results.length} results.`;
+      messageText = _(`Your search for "${query}" returned ${results.length} results.`);
```

Comment:
Yeah definitely - we should deal with that when we add something that does this job properly. Since I don't know what we're using I glossed over it for now.